README.md

```diff
@@ -49,34 +49,35 @@ Our builds can be fetched from `kinvolk.io/aci/rkt/stage1-kvm`, e.g.
 rkt image fetch --insecure-options=image kinvolk.io/aci/rkt/stage1-kvm:1.23.0,kernelversion=4.9.6
```

Comment:
while you are at it, I would like if there was an example of "rkt run ..... uname -a" command directly in the readme